pkg/flashdb/doc.txt

```diff
@@ -0,0 +1,71 @@
+/**
+ * @defgroup pkg_flashdb    FlashDB: Database for Flash
```

Comment:
might be better to go with 

```suggestion
 * @defgroup pkg_flashdb   flashdb: Database for Flash
```

but this is better then most modules so this comment might as well be ignored.

https://github.com/RIOT-OS/RIOT/issues/7094 < this is an age old RFC that is discussing the these goup titles

Reply:
But that’s the name of the software - and the needed module names are explained just below that.